src/payments/payments.controller.ts

```
@@ -64,10 +62,7 @@ export class PaymentsController {
     type: PaymentStatusResponseDto,
   })
   @ApiNotFoundResponse({ description: 'Payment intent not found' })
-  async getPaymentStatus(
-    @Param('id', ParseUUIDPipe) paymentIntentId: string,
-  ) {
+  async getPaymentStatus(@Param('id', ParseUUIDPipe) paymentIntentId: string) {
```

ESLint

[error] Replace `'id'` with `"id"`

(prettier/prettier)